app/src/main/java/com/kickstarter/ui/views/compose/checkout/PaymentSchedule.kt

```diff
@@ -390,5 +390,6 @@ fun StatusBadge(state: PaymentIncrementState, stateReason: PaymentIncrementState
         }
         PaymentIncrementState.UNKNOWN__ -> {}
         PaymentIncrementState.CHARGEBACK_LOST -> {}
+        PaymentIncrementState.REFUNDED -> {}
```

Comment:
@jlplks ack for this piece, had to do it for the app to compile after the schema update